src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.cs

```diff
@@ -84,7 +86,7 @@ public void Flush()
             }
         }
 
-        public void FlushIfBufferThresholdReached()

```

Comment:
Set this to `private` cause there was no reason for it to be public. It's not a breaking change because the class is internal.